packages/theming/apple-theme/src/appleColors.macos.ts

```diff
@@ -282,6 +283,9 @@ export function fallbackApplePalette(): ThemeColorDefinition {
   const fluentUIApple = getFluentUIApplePalette();
   const applePlatform = getAppleSemanticPalette();
 
+  // GH:## Until RN-SVG Supports PlatformColor, we need this workaround
+  const neutralForeground3 = Appearance.getColorScheme() === 'light' ? '#272727' : '#FFFFFF';
+
```

Comment:
There's a util we separated that's better for this now, right @rurikoaraki ?

Reply:
Yeah getCurrentAppearance() from theming-utils

Reply:
Thanks! I have switched to using that =)